metatron/src/main/java/uk/gov/ida/eidas/metatron/MetatronConfiguration.java

```diff
@@ -1,7 +1,18 @@
 package uk.gov.ida.eidas.metatron;
 
+import engineering.reliability.gds.metrics.config.PrometheusConfiguration;
 import io.dropwizard.Configuration;
 
-public class MetatronConfiguration extends Configuration {
+public class MetatronConfiguration extends Configuration implements PrometheusConfiguration {
 
+    private String countriesConfig;
+
+    @Override
+    public boolean isPrometheusEnabled() {
+        return true;
```

Comment:
Should this be over-rideable, for running locally and what-not?

Reply:
Well, strictly speaking since I took the Prometheus thing out (as it was in verify-saml-libs anyway), it's not needed at all but it's a straight copy from somewhere else and if Prometheus is enabled, it just adds a servlet to the container so wouldn't I think affect any thing running locally.

Reply:
The Prometheus bundle is still being loaded in the Application class?

Reply:
It's going now that the Prometheus bundle is going (it's a config requirement of the PrometheusBundle)